proto/okapi/hashing/v1/hashing.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package okapi.hashing.v1;
+
+option csharp_namespace = "Okapi.Hashing.V1";
+option go_package = "github.com/trinsic-id/okapiproto";
+option java_package = "trinsic.okapi.hashing.v1";
+
+
+message Blake3HashRequest {
+    bytes data = 1;
+    optional bytes key = 2;
```

Comment:
I would simply use a different message `Blake3KeyedHashRequest` instead setting this field `optional`. From an API self-description, this field is "unused" in the first case, and "required" in the second.